mdDocs/faq.md

```diff
@@ -299,3 +299,37 @@ public var musicTrack: MediaTrack? { get }
 ...
 }
 ```
+
+### 15. I want to check whether my token is expired.
+
+Available from **'1.0.18'** version
```

Comment:
Starting from **'1.0.18'** version it is available to check if token is expired.